samples/bluetooth/mesh/dfu/target/README.rst

```diff
@@ -186,6 +184,9 @@ Logging
 
 In this sample, UART and SEGGER RTT are available as logging backends.
 
+.. note::
+   With :ref:`zephyr:nrf52840dongle_nrf52840`, only logging over UART is available.
```

Comment:
Why not over USB? I guess it is quite awkward to use UART on USB dongle.

Reply:
This edit was requested by @PavelVPV, in the comment above.

Reply:
Dongle compiles with USB CDC ACM by default.

Reply:
does UART mean here virtual UART (USB CDC)?

Reply:
Log backend uses UART driver API, USB CDC ACM implements UART driver.

Reply:
I don't follow. Is any change needed?

Reply:
I also forgot to refresh my browser before asking that! No problem then :)